cms/api.py

```diff
@@ -533,7 +536,8 @@ def publish_page(page, user, language):
 
         For publishing functionality see `djangocms-versioning: <https://github.com/django-cms/djangocms-verisoning>`_
     """
-    warnings.warn('This API function has been removed', UserWarning)
+    warnings.warn('This API function has been removed. For publishing functionality use djangocms-versioning.',
```

Comment:
```suggestion
    warnings.warn('This API function has been removed. For publishing functionality use a package that adds publishing, such as: djangocms-versioning.',
```

Slight reword to state that djangocms-versioning is not the only package you can use.